api/comics/management/commands/seed.py

```diff
@@ -0,0 +1,48 @@
+from django.core.management.base import BaseCommand
+import random
+from comics.models import Comic
+import os, sys
+from . import data
+import logging
+
```

Comment:
```suggestion
from faker import Faker

fake = Faker()
```
Adding this will let us generate fake text for the description